cinch/roles/nginx/files/etc/nginx/conf.d/default.conf

```diff
@@ -0,0 +1,44 @@
+#server {
```

Comment:
Why this is file commented out?  If it needs to be this way by default, perhaps add a comment at the top explaining why.